### PR TITLE
Update protonvpn.sh

### PR DESCRIPTION
**Pull request by loicyannou**
The downloadURL has changed from "https://protonvpn.com/download" to "https://protonvpn.com/download-macos"

### CODE REVIEW

**Comment by loicyannou** [comment]
Actually, it seems that the curl does not return the correct dmg version (1.9.6 instead of 3.3.3) and I can't figure out why.
I need some help, please

**Comment by guy-tallon** [comment]
https://protonvpn.com/download/macos-update3.xml seems to hold the version

So `curl -s "https://protonvpn.com/download/macos-update3.xml" | grep -o 'https*://[^"]*\.dmg' | head -1` should work

**Comment by guy-tallon** [comment]
```bash
protonvpn)
    name="ProtonVPN"
    type="dmg"
    downloadURL=$(curl -s "https://protonvpn.com/download/macos-update3.xml" | grep -o 'https*://[^"]*\.dmg' | head -1)
    appNewVersion=$(echo $downloadURL | sed -e 's/^.*\/Proton.*_v\([0-9.]*\)\.dmg/\1/g')
    expectedTeamID="J6S6Q257EK"
    ;;
```

**Comment by jnichols35** [comment]
The more I go down this rabbit hole, the more confusing it becomes. 
I thought I would check out which version was on [winget](https://github.com/microsoft/winget-pkgs/tree/8664856543abd9540665e7b9e96e750b147da81d/manifests/p/ProtonTechnologies/ProtonVPN/3.2.7) and it's the same as on their [github](https://github.com/ProtonVPN/win-app/releases/tag/3.2.7): 3.2.7

But the [download link](https://protonvpn.com/download-windows) on their site is for: 2.4.3

Basically I was hoping the [ios/mac repo](https://github.com/ProtonVPN/ios-mac-app) could have been used as a source of truth but its on version 3.1.4. 

So really nothing makes sense

**Comment by guy-tallon** [comment]
Haha, yes it's pretty all over the place for macOS. Odd as the other ones are unto date in GitHub.
I guess they've settled on the rss feed

**Comment by guy-tallon** [comment]
or maybe this

```bash
protonvpn)
    name="ProtonVPN"
    type="dmg"
    downloadURL=$(curl -s "https://protonvpn.com/download/macos-update3.xml" | xmllint --xpath 'string(//enclosure/@url)' -)
    appNewVersion=$(echo $downloadURL | sed -e 's/^.*\/Proton.*_v\([0-9.]*\)\.dmg/\1/g')
    expectedTeamID="J6S6Q257EK"
    ;;
```
@loicyannou Do you want to make another commit?

**Comment by loicyannou** [comment]
Sorry @guy-tallon, I missed your message
Thanks for your help
I updated the commit with your insights

**Comment by guy-tallon** [comment]
Great! Thank you for updating it

**Comment by BigMacAdmin** [comment]
Thank you!
